scripts/zones/Yuhtunga_Jungle/mobs/Carthi.lua

```diff
@@ -15,18 +15,18 @@ end;
 -----------------------------------
 
 function onMobDeath(mob, player, isKiller)
+    if (isKiller) then -- Causes this to only run once per Carthi kill
```

Comment:
One thing I'm not so sure of is the way this is requiring both Nm kills, I'd wager retail lets you depop one and then when the 2nd dies it only cares that both are now dead. But I only set out to quick get this back to working near as it had been as fast as I could, not redo whatever the previous implementation might have been wrong about. If anyone wants to PR a better implementation (maybe ditch the server vars..) I'll happily close this PR in favor of theirs.